api/python/provisioner/commands/check.py

```diff
@@ -385,6 +385,9 @@ class Check(CommandParserFillerMixin):
 
     _PRV_METHOD_MOD = "_"  # private method modificator
 
+    STONITH = "stonith"
+    ACCESSIBLE = "accessible"
+
```

Comment:
We can even move this to `config.py` and address as `cfg.STONITH` , so we can handle all validation-related config values in a single place.

Reply:
I think it is unnecessary:

1. These values are only needed by the entire logic of the `Check` class and I don't see how we can reuse them outside of `check.py`
2. Also, we have already two different methods that provide bmc checks without these parameters: `_bmc_stonith` and `_bmc_accessibility`.

So these parameters help us to reuse `_validate_bmc_check` code without duplication. But if we really need them outside of check.py, I agree, we need to move it to `config.py`

Reply:
I agree with @dmitrydb , These are just constants we used in Check class actual check names are already in config so I think no need to move them to config.

`BMC_ACCESSIBILITY = "bmc_accessibility"
 BMC_STONITH = "bmc_stonith"
`

Reply:
I agree with the scope not being used. This was more about handling all validation related-parameters in `config.py`.